benches/src/main.rs

```diff
@@ -12,6 +12,7 @@ async fn main() -> eyre::Result<()> {
         erc721::bench().boxed(),
         merkle_proofs::bench().boxed(),
         ownable::bench().boxed(),
+        erc1155::bench().boxed(),
```

Comment:
@qalisander still [doesn't work](https://github.com/OpenZeppelin/rust-contracts-stylus/actions/runs/12064392244/job/33641144702?pr=417) :confused:

Reply:
Yeah. I see..

Reply:
But it works with a second deployment

Reply:
First deployment fails with error

Reply:
And now it seems [work](https://github.com/OpenZeppelin/rust-contracts-stylus/actions/runs/12065754746/job/33645191910?pr=417). Just reduce parallelization level for benchmarks and it got fixed. I hope we will never have this error again!